src/pages/workspace/WorkspaceInviteMessagePage.js

```diff
@@ -78,6 +78,17 @@ class WorkspaceInviteMessagePage extends React.Component {
         };
     }
 
+    componentDidMount() {
+        this.focusTimeout = setTimeout(() => {
+            this.welcomeMessageInputRef.focus();
+            // Below condition is needed for web, desktop and mweb only, for native cursor is set at end by default.
+            if (this.welcomeMessageInputRef.value && this.welcomeMessageInputRef.setSelectionRange) {
+                const length = this.welcomeMessageInputRef.value.length;
+                this.welcomeMessageInputRef.setSelectionRange(length, length);
+            }
```

Comment:
This is caused a regression here https://github.com/Expensify/App/issues/20875#issuecomment-1594421713.

Why do we need this?

Reply:
cc: @mollfpr @PrashantMangukiya

Reply:
Why is https://github.com/Expensify/App/issues/20875 a regression of this?

On the native the cursor is set at the end of the text, so we want the same behavior for the web and desktop.